set.mm

```diff
@@ -810491,6 +810542,678 @@ as an isomorphism in the category of graphs (something like "GraphIsom
+    $( Graph local isomorphism is transitive.  (Contributed by AV,
+       10-Jun-2025.) $)
+    grlictr $p |- ( ( R ~=lgr S /\ S ~=lgr T ) -> R ~=lgr T ) $=
+      ( vf vr vg vh vs wbr wa cvv wcel cfv cv cclnbgr co cisubgr cgric oveq2d
+      wi cgrlic grlicrcl anim12i cvtx wf1o wral wex eqid grilcbri ccom vex coex
+      a1i f1oco ad2ant2r f1of ffvelcdmda oveq2 fveq2 breq12d rspcv syl wf fvco3
+      sylan eqcomd breq2d sylibd ex com3r imp31 anim1ci grictr ralimdva expimpd
+      wceq adantl imp f1oeq1 fveq1 ralbidv anbi12d spcedv exlimiv syl2an adantr
+      jca com12 wb dfgrlic2 ad2ant2rl mpbird mpdan ) ABUAIZBCUAIZJZAKLZBKLZJZWR
+      CKLZJZJZACUAIZWNWSWOXABAUBCBUBUCWPXBJXCAUDMZCUDMZDNZUEZAAENZOPQPZCCXHXFMZ
+      OPZQPZRIZEXDUFZJZDUGZWPXPXBWNXDBUDMZFNZUEZXIBBXHXRMZOPZQPZRIZEXDUFZJZFUGZ
+      XQXEGNZUEZBBHNZOPZQPZCCYIYGMZOPZQPZRIZHXQUFZJZGUGZXPWOEFABXDXQXDUHZXQUHZU
+      IHGBCXQXEYTXEUHZUIYFYRXPYEYRXPTFYRYEXPYQYEXPTGYQYEXPYQYEJZXOXDXEYGXRUJZUE
+      ZXICCXHUUCMZOPZQPZRIZEXDUFZJDKUUCUUCKLUUBYGXRGUKFUKULUMUUBUUDUUIYHXSUUDYP
+      YDXDXQXEYGXRUNUOYQYEUUIYPYEUUITYHYPXSYDUUIYPXSJZYCUUHEXDUUJXHXDLZJZYCUUHU
+      ULYCJYCYBUUGRIZJUUHUULUUMYCYPXSUUKUUMXSUUKYPUUMXSUUKYPUUMTXSUUKJZYPYBCCXT
+      YGMZOPZQPZRIZUUMUUNXTXQLYPUURTXSXDXQXHXRXDXQXRUPZUQYOUURHXTXQYIXTVPZYKYBY
+      NUUQRUUTYJYABQYIXTBOURSUUTYMUUPCQUUTYLUUOCOYIXTYGUSSSUTVAVBUUNUUQUUGYBRUU
+      NUUPUUFCQUUNUUOUUECOUUNUUEUUOXSXDXQXRVCUUKUUEUUOVPUUSXDXQXHYGXRVDVEVFSSVG
+      VHVIVJVKVLXIYBUUGVMVBVIVNVOVQVRWGXFUUCVPZXGUUDXNUUIXDXEXFUUCVSUVAXMUUHEXD
+      UVAXLUUGXIRUVAXKUUFCQUVAXJUUECOXHXFUUCVTSSVGWAWBWCVIWDWHWDVRWEWFXBXCXPWIZ
+      WPWQWTUVBWRWREDACXDXEKKYSUUAWJWKVQWLWM $.
+  $}
+
+
+  ${
+    $d V x y z $.  $d .~ x y z $.
+    brinxper.r $e |- ( x e. V -> x .~ x ) $.
+    brinxper.s $e |- ( x e. V -> ( x .~ y -> y .~ x ) ) $.
+    brinxper.t $e |- ( x e. V -> ( ( x .~ y /\ y .~ z ) -> x .~ z ) ) $.
+    $( Conditions for a reflexive, symmetric and transitive binary relation to
+       be an equivalence relation over a class ` V ` .  (Contributed by AV,
+       11-Jun-2025.) $)
+    brinxper $p |- ( .~ i^i ( V X. V ) ) Er V $=
+      ( cv wbr wa wcel wi brxp adantr sylbi impcom brin com12 adantl cxp sylbb2
+      cin relinxp ancom jctird 3imtr4i expd simplr simprl anim12ci exp31 anbi2i
+      imp jca 3imtr4g sylibr id sylanbrc simplbi impbii bitr4i iseri ) ABCEDEEU
+      AZUCZEEDUDAIZBIZDJZVFVGVDJZKZVGVFDJZVGVFVDJZKZVFVGVEJZVGVFVEJVIVHVMVIVFEL
+      ZVGELZKZVHVMMVFVGEENZVQVHVKVLVOVHVKMVPGOVQVPVOKVLVOVPUEVGVFEENUBUFPQVFVGD
+      VDRZVGVFDVDRUGVNVGCIZVEJZKVFVTDJZVFVTVDJZKZVFVTVEJVNWAWDVNVJWAWDMVSWAVJWD
+      WAVGVTDJZVGVTVDJZKZVJWDMVGVTDVDRWGVHVQKZWBVOVTELZKZKZVJWDWFWEWHWKMZWFVPWI
+      KZWEWLMVGVTEENWMWEWHWKWMWEKZWHKWBWJWNWHWBWEWHWBMWMWHWEWBVQVHWEWBMZVOVHWOM
+      VPVOVHWEWBHUHOQSTUNWNWIWHVOVPWIWEUIVHVOVPUJUKUOULPQVIVQVHVRUMWCWJWBVFVTEE
+      NUMUPPSPUNVFVTDVDRUQVOVFVFDJZVFVFVDJZKZVFVFVEJVOWRVOWPWQFVOVOVOWQVOURZWSV
+      FVFEENZUSUOWQVOWPWQVOVOWTUTTVAVFVFDVDRVBVC $.
+  $}
```

Comment:
This is a generic theorem, it could be moved to a section specific to relations.